src/components/Link/Link.js

```diff
@@ -3,21 +3,35 @@ import PropTypes from 'prop-types';
 import classnames from 'classnames';
 import style from './Link.scss';
 
-const Link = props => (
-  <a
-    className={classnames(style.link, props.className)}
-    href={props.href}
-    rel="noopener noreferrer"
-    target="_blank"
-  >
-    {props.children || props.href}
-  </a>
-);
+const Link = (props) => {
+  const classes = classnames(
+    {
+      [style.link]: !props.unstyled,
+    },
+    props.className,
+  );
+
+  return (
+    <a
+      className={classes}
+      href={props.href}
+      rel="noopener noreferrer"
+      target="_blank"
+    >
+      {props.children || props.href}
+    </a>
+  );
+};
 
 Link.propTypes = {
+  unstyled: PropTypes.bool,
   children: PropTypes.string,
   className: PropTypes.string,
   href: PropTypes.string.isRequired,
 };
 
+Link.defaultProps = {
+  unstyled: false,
```

Comment:
Defaulting a prop to something falsey is redundant, but I'll allow it. (Unless you want to remove it and add a style section to the readme and put this in there as our very first rule.)